app/controllers/api/v1/auth_sessions_controller.rb

```diff
@@ -1,41 +1,176 @@
 class Api::V1::AuthSessionsController < Devise::SessionsController
+  include RackSessionsFix
+  require 'jwt'
+
+  respond_to :json
+
   protect_from_forgery with: :null_session
-  before_action :sign_in_params, only: :create
-  before_action :load_user, only: :create
+  #before_action :sign_in_params, only: :create
+  #before_action :load_user, only: :create
+  #skip_before_action :verify_authenticity_token, :only => :create
 
   # sign in
   def create
-    if @user.valid_password?(sign_in_params[:password])
-      sign_in "user", @user
+# -- remove for JWT conversion ---
+#    if @user.valid_password?(sign_in_params[:password])
+#      sign_in "user", @user
+#      render json: {
+#        messages: "Signed In Successfully",
+#        is_success: true,
+#        data: {user: @user}
+#      }, status: :ok
+#    else
+#      render json: {
+#        messages: "Signed In Failed - Unauthorized",
+#        is_success: false,
+#        data: {}
+#      }, status: :unauthorized
+#    end
+#  end
+    #logger.info "login: params: #{params.inspect}"
```

Comment:
I'd really like to avoid all of this commented code. Again, if we're using it soon (going to turn right back around and uncomment it), that's one thing.. but I'd like to avoid it as much as possible.